simphony_ui/tests/test_ui.py

```diff
@@ -0,0 +1,32 @@
+"""
+Tests user interface
+"""
+
+import unittest
+import simphony_ui.ui as ui
+from traits.api import TraitError
+
+
+class TestModel(ui.HasStrictTraits):
+
+    pos_float = ui.PositiveFloat(1e-6)
+    pos_int = ui.PositiveInt(10)
+
+
+class TestUI(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
you want a fresh object at every new test, otherwise if one test goes wrong, all the ones following will go wrong as well. use setUp()